portfolio/src/main/webapp/script.js

```diff
@@ -13,11 +13,14 @@
 // limitations under the License.
 
 /**
- * Toggles the drop down text.
+ * Toggles the dropdown text and dropdown arrow.
  * @param {string} id The id of the dropdown content to be toggled.
+ * @param {!Event} event The click event on a dropdown bar.
```

Comment:
```suggestion
 * @param {!Event} event The click event for the dropdown bar.
```

Reply:
Should it be dropdown content?

Reply:
I mean that the click is on the dropdown bar?

Reply:
Yes :)